tests/unit/explore_test.sh

```diff
@@ -67,7 +67,12 @@ function test_explore_files_under_git_repo()
   assertEquals "($LINENO)" "$MSG_OUT" "$output"
 
   output=$(explore_main 'GNU grep' '.' 'TEST_MODE')
-  expected_result="git grep -e 'GNU grep' -nI ."
+  expected_result="git grep -C 0 -e 'GNU grep' --line-number -I ."
+  assertEquals "($LINENO)" "$expected_result" "$output"
+
+  # Test for non zero context value
+  output=$(explore_main --show-context 5 'GNU grep' '.' 'TEST_MODE')
+  expected_result="git grep -C 5 -e 'GNU grep' --line-number -I ."
```

Comment:
This type of test is okay, but I would prefer something that uses the feature. For example, it would be better to search for something in one of the sample files, capture the output, and check if it worked as expected.